cardano-testnet/src/Testnet/Util/Runtime.hs

```diff
@@ -67,16 +72,16 @@ data NodeRuntime = NodeRuntime
 
 data PoolNode = PoolNode
   { poolRuntime :: NodeRuntime
-  , poolKeys :: PoolNodeKeys
+  , poolKeys :: PoolNodeKeys -- used in Alonzo LeadershipShedule
   }
 
 data PoolNodeKeys = PoolNodeKeys
-  { poolNodeKeysColdVkey :: FilePath
+  { poolNodeKeysColdVkey :: FilePath -- == Operator keys -- used in LeadershipShedule
```

Comment:
I'm partial to using something more type safe than `FilePath` for each type of file to make usage information easier to find but it doesn't have to be this PR.